test/requirements_aistore.txt

```diff
@@ -1,2 +1,2 @@
+aistore == 1.0.2
```

Comment:
Question: what makes you choose `==` instead of `>=`?

Reply:
It seems like we are anticipating more changes to AIStore's API in the future? Is that correct?

Is the plan to increase the version number here and adjust the our usages of the APIs as AIStore's APIs change?

Reply:
Hi @NivekT, all versions after 1.0.2 should work. We didn't want the torch/data pipeline ever to fail if in case we were trying some experimental changes on our side. 
But we will take care of it. We changed requirements to always fetch the latest version as we don't want people to spend time resolving version conflicts.